tests/tests.py

```diff
@@ -19,20 +18,23 @@ def setUp(self):
         articles_cnt = 4
         users_cnt = 10
 
-        def mock_process(self):
+        def mock_process(self=None):
```

Comment:
In python 2, it's needed because:
DataParser.process() will correspond to mock_process(DataParser)

In Python 3, it's not needed because:
DataParser.process() will correspond to mock_process() (that's why I'm using a default value)